infra/gcp/clusters/projects/k8s-infra-public-pii/main.tf

```diff
@@ -0,0 +1,118 @@
+
+
+/**
+ * Copyright 2021 The Kubernetes Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  project_id  = "k8s-infra-public-pii"
+  bucket_name = "k8s-infra-artifacts-gcslogs"
+  dataset-id  = replace(local.bucket_name, "-", "_")
+}
+
+
+data "google_billing_account" "account" {
+  billing_account = "018801-93540E-22A20E"
+}
+
+data "google_organization" "org" {
+  domain = "kubernetes.io"
+}
+
+
+resource "google_project" "project" {
+  name            = local.project_id
+  project_id      = local.project_id
+  org_id          = data.google_organization.org.org_id
+  billing_account = data.google_billing_account.account.id
+}
+
+resource "google_project_service" "project" {
+  project = google_project.project.id
+
+  for_each = toset([
+    "bigquery.googleapis.com",
+    "bigqueryreservation.googleapis.com",
+    "bigquerytransfer.googleapis.com",
+    "storage-component.googleapis.com"
+  ])
+
+  service = each.key
+}
+
+// Ensure k8s-infra-gcp-auditors@kubernetes.io can administer all the Biquery
+// resources except the datasets and the datasets tables
+resource "google_project_iam_member" "auditors-bq-resources-admins" {
+  project = google_project.project.id
+  role    = "roles/bigquery.resourceAdmin"
+  member  = "group:k8s-infra-gcp-auditors@kubernetes.io"
+}
```

Comment:
I might be wrong, but I'm not sure I want a traditionally read-only group to be able to create things like capacityCommitments or reservationAssignments.

They should already have all the necessary permissions to export metadata (like which datasets exist, and who's able to access them) via the `audit.viewer` role assigned at the organization level.  They should not be able to view the data within the datasets (same mindset with buckets: they are not granted read access to all objects in all buckets, but they are granted list access)
```suggestion
```

ref: https://cloud.google.com/bigquery/docs/access-control

Reply:
Removed.